testcases/kernel/syscalls/lgetxattr/lgetxattr02.c

```diff
@@ -83,12 +96,18 @@ static void setup(void)
 	}
 }
 
+void cleanup(void)
+{
+        remove("testfile");
+        remove("symlink");
+}
+
 static struct tst_test test = {
-	.needs_tmpdir = 1,
```

Comment:
Please add comment here why we are removing this and ehat happens if we don't remove. How this is related to using root fs?

Reply:
Done